portfolio/src/main/webapp/script.js

```diff
@@ -46,18 +46,30 @@ function addRandomQuote() {
 }
 
 function getComments() { //examples/stats-server
-    fetch('/data').then(response => response.json()).then((strings) => {
-        const commentListElement = document.getElementById('comment-container');
-        commentListElement.innerHTML = '';
-        strings.forEach((line) => {
-            commentListElement.appendChild(createListElement(line));
-        });
+    var loggedIn = false;
+    fetch('/login').then(response => response.json()).then((bool) =>{
```

Comment:
nit: `bool` is not a good name for a parameter. Besides, you are already assigning `bool` to another name `loggedIn` right at the beginning of this function body, so you could just use `loggedIn` instead of `bool` in this line (of course you should remove the next line in that case).